{
    "config": {
        "error": {
            "cannot_connect": "Falha ao conectar",
            "invalid_auth": "Autentica\u00e7\u00e3o inv\u00e1lida",
            "unknown": "Erro inesperado"
        },
        "step": {
            "auth": {
                "data": {
                    "password": "Senha",
                    "username": "Usu\u00e1rio"
                }
            },
            "user": {
                "data": {
                    "host": "Nome do host"
                }
            }
        }
    }
}